frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -25,16 +30,20 @@ export type ToolbarOverflowMenuProps = {
  *  @property {boolean} isOpen - whether the component will be open
  *  @property {boolean} isEmojiGridOpen - whether the emoji grid will be open
  *  @property {Dispatch<SetStateAction<boolean>>} setIsEmojiGridOpen - toggle whether the emoji grid is shown or hidden
- *  @property {RefObject<HTMLButtonElement | null>} anchorRef - the button ref for the menu
- *  @property {() => void} closeMenu - hides the menu when user clicks away from the menu
+ *  @property {(event?: MouseEvent | TouchEvent) => void} closeMenu - hides the menu when user clicks away from the menu
+ *  @property {Function} toggleShareScreen - toggles the user's screenshare
+ *  @property {boolean} isSharingScreen - whether the user is sharing their screen
+ *  @property {number} toolbarButtonsDisplayed - total buttons displayed on the toolbar
```

Comment:
Nit: "total buttons" -> "number of buttons"?  Also, is the convention to prefix counts with `numOf`?  or `num`?

Reply:
We haven't been following that in this repo 😬 . We seem to use `someElementsCount`, so I changed it to be more consistent.